Claude-fable-5-best-practices.md

```
@@ -0,0 +1,522 @@
+---
+
+## 目次
+
+1. [Claude Fable 5 とは何か](#1-claude-fable-5-とは何か)
+2. [タイムライン: リリースから輸出規制、復旧まで](#2-タイムライン-リリースから輸出規制復旧まで)
+3. [安全分類器と自動フォールバックの仕組み](#3-安全分類器と自動フォールバックの仕組み)
+4. [プロンプティング思想の転換: チェックリストからゴールへ](#4-プロンプティング思想の転換-チェックリストからゴールへ)
+5. [Effort(推論深度)レベルの使い方](#5-effort推論深度レベルの使い方)
+6. [Claude Code での実践設定](#6-claude-code-での実践設定)
+7. [Loop Engineering: 長時間自律ループの設計思想](#7-loop-engineering-長時間自律ループの設計思想)
+8. [Thariq の「Unknowns フレームワーク」(参照ポスト解説)](#8-thariq-の-unknowns-フレームワーク参照ポスト解説)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)